### PR TITLE
Force SSL certs for RHEL

### PR DESCRIPTION
**Pull request by zfeldstein**
this also sets up the base venv for all OS's.

### CODE REVIEW

File: roles/common/tasks/ssl.yml [hunk 1]
```diff
@@ -76,13 +76,26 @@
...
-  when: os == 'rhel'
+  when:
+    - os == 'rhel'
+    - openstack_install_method != "distro"
+
+- name: force our ssl cert for python libs on rhel
+  file: src=/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt dest={{ item }} owner=root
+        mode=0644 state=link force=yes
+  with_items:
+    - /opt/openstack/base/lib/python2.7/site-packages/requests/cacert.pem
```
Comment (nirajdp76) [inline roles/common/tasks/ssl.yml]: shouldn't we be using basevenv_lib_dir variable
also, seems like we have facts or variable defined for ca-bundle, etc.

**Comment by nirajdp76** [comment]
@paulczar can u please review and merge this.

File: roles/common/tasks/python.yml [hunk 2]
```diff
@@ -27,51 +27,48 @@
...
-    until: result|succeeded
-    retries: 5
+- name: upgrade setuptools for base venv (osp)
+  pip:
+    name: "setuptools"
+    virtualenv: "{{ common.python.base_venv }}"
+    extra_args: "{{ pip_extra_args }} --upgrade"
+  register: result
+  until: result|succeeded
+  retries: 5
```
Comment (scoop206) [inline roles/common/tasks/python.yml]: should the '(osp)'  be removed from the above and below action names ?

Comment (paulczar) [inline roles/common/tasks/python.yml]: yeah it should be.  good catch.

**Comment by nirajdp76** [comment]
retest

**Pull request merged**